packages/actor-scraper/playwright-scraper/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM apify/actor-node:16 AS builder
```

Comment:
we had a discussion about the dockerfiles yesterday with @mnmkng, we should generally use the same image for the builder, as it will have the browser already preinstalled (and we will pull that image later on anyway)

there might be more improvements, but lets do at least this one thing, we can port the rest of the changes later

Reply:
maybe instead of having multistage in this case - we could just remove src folder? What's the point of double installation? Well - dev dependencies will stay, but is it that bad really?

Reply:
Also, wasn't there some stuff about the access right for node vs browser images? Or should it just work?

Reply:
@B4nan  So I tried to build it from local package files (under my acc, when I was checking whether it's working at all), and it failed, to make sure I tried to build the template actor using the same docker for both stages. Did I get your comment wrong or it's expected?

Reply:
Dev deps are the point, and yes, it is bad, or better say unnecessary. Multistage builds are the correct solution.

But you are right that this might need more changes because of the different user in browser images. We were discussion that too, IMHO we should not change user in base images, that belongs to the project's dockerfile, not to those it extends... 

I guess lets revert it now and get back to it after we have a proper fix elsewhere we can just port in here.